css/index.css

```diff
@@ -1,3 +1,287 @@
+}
+
+.navigation a{
+    display: flex;
+	text-decoration: none;
+	color: #989DA5;
+}
+
+.navigation a:hover {
+	color: black;
+	font-size: 20px;
+	border-top: 2px dotted #989DA5;
+}
+
+.navigation .logo {
+	padding: 10px 20px;
+	margin: 0 auto;
+}
+
+/* sticky nav effect.........
+.stuck{
+	background: white;
+	position: fixed;
+	top: 0;
+	z-index: 1;
+	width: 100%;
+} */
+
+
+/* syling header.............. */
+
+.intro-header{
+	display: flex;
+	align-items: center;
+	justify-content: space-around;
+	width: 100%;
+	/* height: auto; */
+	/* margin: 0 auto; */
+	padding: 40px;
+	overflow: hidden;
+	border-bottom: 1px solid black;
+}
+
+.block1{
+	padding: 60px;
+	margin: 40px 20px;
+	text-align: center;
+	/* box-sizing: border-box; */
+}
+
+.block1 h1{
+	font-family: 'Gugi', cursive;
+	font-size: 70px;
+	padding-bottom: 20px;
+}
+
+.block1 a{
+	text-decoration: none;
+	font-size: 15px;
+	padding: 5px 20px;
+	color: black;
+	border: 1px solid black;
+	border-radius: 4px;
+}
+
+.block1 a:hover{
+	color: white;
+	font-size: 20px;
+	background: #989DA5;
+}
+
+.intro-header img{
+	position: relative;
+	margin: 50px;	
+}
+
+@media (max-width: 800px){
+
+    .navigation{
+        display: flex;
+        flex-direction: column-reverse;
+    }
+    
+    .navigation nav {
+        padding-top: 2%;
+    }
+    .intro-header img{
+        display: none;
+    }
+}
+
+/* styling main */
+
+main {
+	display: flex;
+	flex-wrap: wrap;
+	margin: 0 auto;
+	padding-bottom: 20px;
+	border-bottom: 1px solid black;
+}
+main img{
+	width: 100%;
+	margin: 0 auto;
+	padding: 20px;
+}
+.box2{
+	width: 49%;
+	padding: 20px;
+	box-sizing: border-box;
+	margin: 0 auto; 
+}
+
+.box3{
+	width: 33%;
+	margin: 0 auto;
+	padding: 20px;
+}
+
+/* style address & Footer */
+
+address {
+	padding: 20px;
+}
+
+footer{
+	text-align: center;
+	color: #989DA5;
+	padding: 20px;
+}
+
+/* style the service page......... */
+/* .service-main{
+	font-size: 15px;
+}
+
+.service-head h1{
+	padding: 20px 5px;
+	margin: 2px;
+	font-size: 20px;
+}
+.service-head img{
+	width: 100%;
+	margin: 0 auto;
+	position: relative;
+}
+
+.service-main{
+	margin: 20px 0;
+	padding: 20px 0;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-content: space-between;
+	border-top: 1px solid black;
+}
+
+.box-sec{
+	margin: 20px;
+	padding: 20px;
+	width: 450px;
+	height: auto;
+	border: 1px solid black;
+	background-color: rgb(222, 224, 228);
+	border-radius: 5px;
+	/* box-sizing: border-box;
+	overflow: hidden; */
+/* }
+.box-sec p{
+	margin: 5px;
+}
+.box-sec button{
+	text-decoration: none;
+	margin: 5px;	
+	padding: 5px 25px;
+	color: black;
+	background: white;
+	border: 1px solid black;
+	border-radius: 3px solid black;
+}
+
+.final-sec{
+	padding: 20px 0;
+	display: flex;
+	justify-content: space-evenly;
+}
+
+.final-sec img{
+	flex-basis: 50%;
+	margin: 20px 20px;
+}
+
+.box-final{
+	display: flex;
+	flex-direction: column;
+	margin: 20px 20px;
+	padding: 2%;
+} */ */
+
+
 
 
```

Comment:
here is the media queries:
`/* Tablet */
@media (max-width: 800px) {

    .container {
        width: 100%;
    }

    header {
        flex-wrap: wrap;
    }
  
    header nav {
        width: 768px;
        padding: 0 30px;
    }

    header .logo {
        order: -1;
        display: block;
        margin: 30px auto 0;
    }

    .cta {
        justify-content: center;
    }
    
    .cta .cta-text {
        margin: 0;
    }

    .cta img {
        display: none;
    }

    .main-content .middle-img {
       width: 100%;
    }

    .main-content {
       margin: 75px 20px 35px;
    }

    .text-content {
        margin: 0 10px;
    }

    .contact {
        margin: 0 30px;
    }
}

/* Phone */
@media (max-width: 500px) {

    header {
        justify-content: center;
    }

    header nav {
        width: 100%;
        flex-direction: column;
        margin: 0;
        padding: 0;
    }

    header nav a {
        width: 100%;
        display: block;
        text-align: center;
        padding: 20px 0;
        font-size: 20px;
        border-bottom: 1px solid gray;
    }

    header nav a:first-child {
        border-top: 1px solid gray;
    }

    header nav a:hover {
        background: gray;
    }

    header .logo {
        margin: 20px 0;
    }

    .cta {
        margin-top: 50px;
        flex-flow: column;
        align-items: center;
    }

    .cta img {
        max-width: 80%;
    }

    .main-content .top-content {
        justify-content: space-around;
        flex-direction: column;
        height: 300px;
    }

    .main-content .bottom-content {
        flex-wrap: wrap;
        align-content: space-between;
        height: 440px;
    }

    .text-content {
        margin: 0;
    }

    .contact {
        margin: 20px;
    }
}
`